src/main/java/core/basesyntax/database/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.database;
+
+import core.basesyntax.service.Product;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static Map<Product, Integer> products = new HashMap<>();
+
+    public static Map<Product, Integer> getProducts() {
+        return products;
+    }
```

Comment:
You can make your Map public and dont call any method. 
```suggestion
    public static Map<Product, Integer> products = new HashMap<>();
```